docs/source/project_create.rst

```diff
@@ -224,14 +292,21 @@ on the first model will be recognized as prior knowledge. Then, a second model
 can be trained on the partly-labeled data, and the new predictions will be
 based on the second model.
 
+.. tip::
+
+  It is suggested to first screen with a simple active learning model (e.g.,
+  the defaults) untill you reach your stopping criteria, then switch to a
+  different model (e.g., doc2vec plus a neural net) and screen again untill
+  you reach your stopping criteria.
 
 Warm up
 =======
 
-In the last step of the setup, step 4, ASReview LAB trains a model and ranks
-the records in your dataset. Depending on the model and the size of your
-dataset, this can take a couple of minutes (or even longer). After the project
-is successfully initialized, you can start reviewing.
+In the last step of the setup, step 4, ASReview LAB runs the feature extracter
+and trains a model and ranks the records in your dataset. Depending on the
+model and the size of your dataset, this can take a couple of minutes (or even
+longer; you can enjoy the `animation video <https://www.youtube.com/watch?v=k-a2SCq-LtA>`_). After the project is successfully
+initialized, you can start reviewing.
```

Comment:
```suggestion
In the last step of the setup, step 4, ASReview LAB runs the feature extractor
and trains a model, and ranks the records in your dataset. Depending on the
model and the size of your dataset, this can take a couple of minutes (or even
longer; you can enjoy the `animation video <https://www.youtube.com/watch?v=k-a2SCq-LtA>`_). After the project is successfully
initialized, you can start reviewing.
```